src/main/java/org/gitlab4j/api/models/MergeRequestFilter.java

```diff
@@ -36,6 +37,8 @@
     private String sourceBranch;
     private String targetBranch;
     private String search;
+    private String in;
+    private MergeRequestFilterWIP wip;
```

Comment:
```wip``` would be better as a Boolean and then simply change it to ```"yes"``` or "```"no"``` when used as I have shown below.